.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,12 @@
+# Task List
+Congratulations! You're submitting your assignment!
+
+## Comprehension Questions
+| Question | Answer    |
+| :------------- | :------------- |
+| Describe what the Model is doing in Rails |  |
+| Describe what the Controller is doing in Rails |    |
+| Describe what the View is doing in Rails |  |
```

Comment:
Maybe asking them to differentiate between the model/view/controller.  This is too direct from the notes.